app/src/main/java/com/fstyle/structure_android/data/source/UserDataSource.java

```diff
@@ -5,25 +5,25 @@
 import com.fstyle.structure_android.data.model.User;
 
 import io.reactivex.Completable;
+import io.reactivex.Flowable;
 import io.reactivex.Maybe;
 import io.reactivex.Single;
 import java.util.List;
+import java.util.Optional;
```

Comment:
ERROR: Unused import: java.util.Optional. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports